src/components/safe-apps/SafeAppList.tsx

```diff
@@ -28,6 +36,24 @@ const SafeAppList = () => {
 
   const debouncedSearchQuery = useDebounce(searchQuery, 500)
 
+  const wcLink: LinkProps['href'] = useMemo(() => {
+    const WC_APP_NAME = 'WalletConnect'
+
+    const wc = allSafeApps.find((app) => app.name === WC_APP_NAME)
+
+    if (!wc) {
+      return ''
+    }
+
+    return {
+      pathname: AppRoutes.apps,
+      query: {
+        safe: router.query.safe,
+        appUrl: wc.url,
+      },
+    }
+  }, [allSafeApps, router.query.safe])
```

Comment:
What's this for?
Even if this is really necessary, we shouldn't search apps by name.

Reply:
I was cautious of hardcoding it because of the `id` changing across chains but it seems like it's the same (and across environments because we don't have multiple). I've channged it accordingly.